src/components/alert/alert.scss

```diff
@@ -89,3 +89,24 @@
     transform: scale(scale.$effect-shrink);
   }
 }
+
+.c-alert__icon {
+  align-items: center;
+  background-color: lighten(color.$base-orange, 30%);
+  border-bottom-left-radius: size.$border-radius-medium;
+  border-top-left-radius: size.$border-radius-medium;
+  color: color.$base-orange;
+  display: flex;
+  grid-area: icon;
+  justify-content: center;
+  width: ms.step(5);
+}
+
+// Adds a box shadow to add the appearance of the alert floating
+// above the containers below. Floating has a white background for higher
+// contrast between the background and the shadow.
+.c-alert__floating {
```

Comment:
This class name is using the element notation (`__`) of the modifier notation (`--`). I think it's meant to be a modifier, in which case the class should be renamed and this style should be moved closer to `c-alert` so its closer to the thing it's modifying.